src/wallet/wallet_state_manager.py

```diff
@@ -197,6 +210,13 @@ async def create(
 
         return self
 
+    def get_derivation_index(self, pubkey: G1Element, max_depth: int = 1000) -> int:
```

Comment:
@Yostra This is a bad way to get derivation index used by rl wallets. I'll delete this and save derivation index in rl_info.